operations/alloy-mixin/alerts/clustering.libsonnet

```
@@ -1,67 +1,97 @@
 local alert = import './utils/alert.jsonnet';
 
-alert.newGroup(
-  'alloy_clustering',
-  [
-    // Cluster not converging.
-    alert.newRule(
-      'ClusterNotConverging',
-      'stddev by (cluster, namespace) (sum without (state) (cluster_node_peers)) != 0',
-      'Cluster is not converging: nodes report different number of peers in the cluster.',
-      '10m',
-    ),
+{
+  newAlloyClusterAlertsGroup(enableK8sCluster=true)::
+    alert.newGroup(
+      'alloy_clustering',
+      [
+        // Cluster not converging.
+        alert.newRule(
+          'ClusterNotConverging',
+          if enableK8sCluster then 
+            'stddev by (cluster, namespace, job) (sum without (state) (cluster_node_peers)) != 0' 
+          else 
+            'stddev by (job) (sum without (state) (cluster_node_peers)) != 0',
+          'Cluster is not converging: nodes report different number of peers in the cluster.',
+          '10m',
+        ),
 
-    alert.newRule(
-      'ClusterNodeCountMismatch',
-      // Assert that the number of known peers (regardless of state) reported by each
-      // Alloy instance matches the number of running Alloy instances in the
-      // same cluster and namespace as reported by a count of Prometheus
-      // metrics.
-      |||
-        sum without (state) (cluster_node_peers) !=
-        on (cluster, namespace, job) group_left
-        count by (cluster, namespace, job) (cluster_node_info)
-      |||,
-      'Nodes report different number of peers vs. the count of observed Alloy metrics. Some Alloy metrics may be missing or the cluster is in a split brain state.',
-      '15m',
-    ),
+        alert.newRule(
+          'ClusterNodeCountMismatch',
+          // Assert that the number of known peers (regardless of state) reported by each
+          // Alloy instance matches the number of running Alloy instances in the
+          // same cluster and namespace as reported by a count of Prometheus
+          // metrics.
+          if enableK8sCluster then 
+          |||
+            sum without (state) (cluster_node_peers) !=
+            on (cluster, namespace, job) group_left
+            count by (cluster, namespace, job) (cluster_node_info)
+          |||
+          else
+          |||
+            sum without (state) (cluster_node_peers) !=
+            on (job) group_left
+            count by (job) (cluster_node_info)
+          |||
+          ,
```

The formatting here seems a bit non-standard:

```suggestion
          if enableK8sCluster then |||
            sum without (state) (cluster_node_peers) !=
            on (cluster, namespace, job) group_left
            count by (cluster, namespace, job) (cluster_node_info)
          ||| else |||
            sum without (state) (cluster_node_peers) !=
            on (job) group_left
            count by (job) (cluster_node_info)
          |||,
```